docs/changehistory/NextVersion.md

```diff
@@ -226,6 +226,8 @@ A [FormatDefinition]($quantity) interface has been added, an extension of Format
 
 [FormatsProvider]($quantity) and [MutableFormatsProvider]($quantity) interfaces and a [SchemaFormatsProvider]($ecschema-metadata) class have been added. This enables quick setup of [FormatterSpec]($quantity) and [ParserSpec]($quantity) to help with display formatting.
 
+In addition, [IModelApp]($core-frontend) now accepts an optional `FormatsProvider` for use across a frontend application, defaulting to an internal implementation that uses [QuantityType]($core-frontend) for now.
```

Comment:
The "for now" is a bit unsettling. Are there plans to change the default at sometime in the future? Can we drop the for now?

Reply:
Yes, we are looking to deprecate QuantityType later in 5.x so the implementation under the hood of that default would change. Let me expand update this section to clarify that